dreamcenter/constants.py

```diff
@@ -82,22 +82,17 @@
     "chair": "chair.png",
     "chair_remains": "chair_remains.png",
     "stats_display": "stats_display.png",
-    "skeleton_dead": "skeleton_dead.png",
-    "spider_dead": "spider_dead.png",
-    "skeleton_damaged": "skeleton_damaged.png",
-    "spider_damaged": "spider_damaged.png",
-    "edward_damaged": "edward_damaged.png",
-    "bg_incorner_mask": "bg_incorner_mask.png",
-    "bg_corner_mask": "bg_corner_mask.png",
+    "bg_incorner_mask": "bg_mask.png",
+    "bg_corner_mask": "bg_mask.png",
 }
 
 # Animation dicts
 ANIMATIONS = {
     "projectile_explode": ["projectile_{:003}".format(frame) for frame in range(1, 3 + 1)],
-    "money_stopped": ["money_{:003}".format(frame) for frame in range(1, 4 + 1)],
-    "skeleton_walk": ["skeleton_walk_{:003}".format(frame) for frame in range(1, 3 + 1)],
-    "skeleton_death": ["skeleton_death_{:003}".format(frame) for frame in range(1, 4 + 1)],
-    "skeleton_stopped": ["skeleton_stopped_{:003}".format(frame) for frame in range(1, 3 + 1)],
+    "money_stopped": ["Money_{:003}".format(frame) for frame in range(1, 4 + 1)],
+    "skeleton_walk": ["Skeleton_Walk_{:003}".format(frame) for frame in range(1, 3 + 1)],
+    "skeleton_death": ["Skeleton_Death_{:003}".format(frame) for frame in range(1, 4)],
```

Comment:
There was no Skeleton_death_004, 3 was the highest

Reply:
asset added